lib/services/iotaRedirector.js

```diff
@@ -248,7 +248,7 @@ function processRequests(req, res, next) {
 
                 callback(parseError, [response, parsedBody]);
             } else {
-                callback(parseError, [response, null]);
+                callback(null, [response, null]);
```

Comment:
It was mentioned by the linter. It didn't fail in the tests. I don't think that's worth mentioning.